code/modules/multiz/level_data.dm

```diff
@@ -108,6 +108,10 @@
 	var/decl/strata/strata
 	///The base material randomly chosen from the strata for this level.
 	var/decl/material/strata_base_material
+	///Strata types to forbid from generating on this level.
+	var/list/forbid_strata = list(
+		/decl/strata/permafrost
```

Comment:
Why is this necessary? Is the maximum temperature check still broken on stable somehow?

Reply:
Oh, right, space levels. Yeah, that checks out. Permafrost kind of sucks, it's just gases and the ways to harvest them are kind of buggy/incomplete.